config/locales/devise.authy.en.yml

```diff
@@ -0,0 +1,24 @@
+en:
```

Comment:
We will need this file in pt-br too, but I can help you with this if you want :]